docs/contribute.rst

```diff
@@ -0,0 +1,20 @@
+.. _contribute:
+
+==================
+Contribute to Docs
```

Comment:
sphinx has a bug where it might compile docs and not create some code blocks, quotes or tables. better view this scripts in github